behave_django/management/commands/behave.py

```diff
@@ -69,6 +81,12 @@ def add_arguments(self, parser):
         """
         Add behave's and our command line arguments to the command
         """
+        usage = "%(prog)s [options] [ [DIR|FILE|FILE:LINE] ]+"
+        description = """\
+        Run a number of feature tests with behave."""
+        parser.usage = usage
+        parser.description = description
```

Comment:
Hmmm, is there a reason why you declare `usage` and `description`, only to assign them immediately afterwards? Would a direct assignment not be good enough? Just wondering why, really.
``` python
        parser.usage = "%(prog)s [options] [ [DIR|FILE|FILE:LINE] ]+"
        parser.description = """\
            Run a number of feature tests with behave."""
```